googlemock/include/gmock/gmock-actions.h

```diff
@@ -570,13 +570,9 @@ class PolymorphicAction {
 
    private:
     Impl impl_;
-
```

Comment:
The original intent is to make `MonomorphicImpl` unassignable, so I think we should explicitly write:
`MonomorphicImpl& operator=(const MonomorphicImpl&) = delete;`
This should match the old semantics.